contrib/Dockerfile

```diff
@@ -29,8 +29,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ADD linux /usr/local/bin
 
-EXPOSE 8080
```

Comment:
Why are we removing this?

Reply:
because it only works for alpha, it doesn't work for Zero